acq4/util/clibrary/CParser.py

```diff
@@ -347,9 +348,9 @@ def removeComments(self, file):
         """Remove all comments from file. (operates in memory; does not alter the original files)"""
         self.assertPyparsing()
         text = self.files[file]
-        cplusplusLineComment = Literal("//") + restOfLine
```

Comment:
These changes seem to make readability suffer in some already dense code.. thoughts? 
Maybe `from pyparsing import (a, bunch, of, specific, stuff)` ?

Reply:
Yeah, that's an additional 4 keystrokes per import to change; I'll go through and hit the noisier modules.

Reply:
I did it for pyparsing and ctypes in a few places. I'll keep an eye out for more opportunities to shorten code this way when it's clear what the libraries in play are.